### ISSUE TITLE
wasmer run-unstable . doesn't work

### ISSUE THREAD

**Issue by syrusakbary** [wasmerio/wasmer#3812]
Running:

```
wasmer run-unstable .
```

Doesn't do anything the wasmer.io website (it just halts)

**Comment by ptitSeb** [comment]
Probably related to #3748

**Comment by Michael-F-Bryan** [comment]
I did some troubleshooting and this seems to have been resolved in #3852.

```console
$ sudo ~/.cargo/bin/wasmer run-unstable --net --mapdir /public:./out .
```

Note that I was originally getting this error because you need to pass in the `--net` flag and use `sudo` to listen on port 80.

```console
$ ~/.cargo/bin/wasmer run-unstable .
2023-05-22T12:17:13.962139Z ERROR static_web_server::server: server failed to start up: failed to bind to [::]:80 address

Caused by:
    Not supported (os error 58)
```
